.github/workflows/performance.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Git Checkout
         uses: actions/checkout@v3
       - name: Set Maven Wrapper
-        run: mvn -N io.takari:maven:0.7.7:wrapper -Dmaven=3.6.3
+        run: mvn -N wrapper:wrapper -Dmaven=3.6.3 --no-transfer-progress
```

Comment:
Now I realize all the calls are like this. `--no-transfer-progress` was added in maven 3.6.1. And here we're specifying 3.6.3. We must have written these lines around the time that maven 3.6.1 was new and github runners didn't have access to that version yet.